platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -11,113 +11,162 @@ For details, visit [Azure Free Account][az-create-account].
 Ensure you have sufficient permissions to create resource groups, an Azure Storage account, and a Batch account.
 :::
 
-## Concepts
+## Azure concepts
 
-### Accounts
+<details>
+  <summary>**Regions**</summary>
 
-Seqera Platform relies on an existing Azure Storage and Azure Batch account. You need at least 1 valid Azure Storage account and Azure Batch account within your subscription.
+  Azure regions are distinct geographic areas that contain multiple data centers strategically located around the world to provide high availability, fault tolerance, and low latency for cloud services. Each region offers a wide range of Azure services — choose a specific region to optimize performance, ensure data residency compliance, and meet regulatory requirements. Azure regions also enable redundancy and disaster recovery options by allowing resources to be replicated across different regions, enhancing the resilience of applications and data.
```

Comment:
I found this very wordy - suggestion to split the first sentence and stay with a single tense.
```suggestion
  Azure regions are specific geographic locations around the world where Microsoft has established data centers to host its cloud services. Each Azure region is a collection of data centers that provide users with high availability, fault tolerance, and low latency for cloud services. Each region offers a wide range of Azure services that can be chosen to optimize performance, ensure data residency compliance, and meet regulatory requirements. Azure regions also enable redundancy and disaster recovery options by allowing resources to be replicated across different regions, enhancing the resilience of applications and data.
```